packages/api-gateway/src/drivers/http/adapters/users/updateUser.js

```diff
@@ -1,51 +1,14 @@
 const errorHandler = require('./errorHandler');
+const getInfoUser = require('./getInfoUser');
 
 async function updateUser(req, reply) {
   try {
-    const {
-      email,
-      avatar,
-      firstName,
-      secondName,
-      firstSurname,
-      secondSurname,
-      birthDate,
-      nationality,
-      dniId,
-      dniFrontImg,
-      dniBackImg,
-      gender,
-      phoneNumber,
-      emergencyNumber,
-      passport,
-      address: {
-        country, city, state, address, zip,
-      },
-    } = req.body;
-
     const data = {
-      ...({ userId } = req.params),
-      email,
-      avatar,
-      firstName,
-      secondName,
-      firstSurname,
-      secondSurname,
-      birthDate,
-      nationality,
-      dniId,
-      dniFrontImg,
-      dniBackImg,
-      gender,
-      phoneNumber,
-      emergencyNumber,
-      passport,
-      address: {
-        country, city, state, address, zip,
-      },
+      ...req.params,
+      ...req.body,
     };
 
-    req.log.info('[http-server]: Update user with: ', data);
+    req.log.info('[http-server]: Update user of: ', getInfoUser(data));
```

Comment:
Looks amazing, my recommendation here is, try to name the functions based on what are doing, if your function is returning an object, what if we name it like `buildUserInfo` or `getUserInfoToLog` or `buildUserInfoToLog`.

Reply:
thanks I'll change it in the next PR